pkg/security/podsecuritypolicy/seccomp/strategy_test.go

```diff
@@ -21,25 +21,31 @@ import (
 	"strings"
 	"testing"
 
+	"k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	api "k8s.io/kubernetes/pkg/apis/core"
 )
 
 var (
-	withoutSeccomp    = map[string]string{"foo": "bar"}
+	foo               = "foo"
+	bar               = "bar"
```

Comment:
I would revert this and use the literal strings... the mental overhead of tracking down the values in the variables here isn't really worth it

Reply:
Reverted that change as suggested. :+1: